metrics-sample-set/metrics-sample-set.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'Decide if the user is in the metrics sample set',
+  startDate: '2015-01-01',
+  subjectAttributes: ['sampleRate', 'uniqueUserId'],
+  independentVariables: ['isInSampleSet'],
+  eligibilityFunction: function (subject) {
+    return true;
+  },
+  groupingFunction: function (subject) {
+    return {
+      isInSampleSet: !! this.bernoulliTrial(subject.sampleRate, subject.uniqueUserId)
```

Comment:
I thought we did not want to pass the sample rate but specify it inside the experiment instead?